### ISSUE TITLE
[Oops.] Font issues in figures

### ISSUE THREAD

**Issue by jeffgerickson** [jeffgerickson/algorithms#124]
There are several minor issues with fonts in the figures.

- Recompile all figures using Apple's broken version of Charter Italic to use older version (see #31)
- Replace SourceSansPro with Roboto for sans-serif text in several figures
- Make sure all important text is at least 5pt (e.g., labels in some graph figures)
- Purge invisible text reported by Acrobat.

See Acrobat's preflight report for details.

**tl;dr: Double-check and recompile _EVERY_ figure.**

**Comment by jeffgerickson** [comment]
(1) Because I'm a perfectionist
(2) Because I want to avoid issues with the print publisher (KDP)
